Extensions/calc.json

```diff
@@ -0,0 +1,333 @@
+{
+  "author": "add_",
+  "description": "A collection of 7 calculations functions to help find slope, gcf and many other values.\n\nEnjoy :)",
```

Comment:
I would stay formal, and remove the enjoy (GDevelop is for everyone, also professionals ;) )